src/v2/Apps/ViewingRoom/Routes/Works/Components/ViewingRoomCarousel.tsx

```diff
@@ -1,42 +1,45 @@
 import React, { useState } from "react"
-import { createFragmentContainer, graphql } from "react-relay"
-import { ViewingRoomCarousel_artwork } from "v2/__generated__/ViewingRoomCarousel_artwork.graphql"
 import { Carousel } from "v2/Components/Carousel"
 import { flowRight } from "lodash"
 
 import {
   Box,
   ChevronIcon,
   Flex,
-  Image,
   ProgressBar,
   breakpoints,
 } from "@artsy/palette"
 
 interface ViewingRoomCarouselProps {
-  artwork: ViewingRoomCarousel_artwork
+  items: any
+  itemRender: any
+  height: any
+  maxWidth?
+  justifyCarousel?
```

Comment:
Similarly, `justifyCarousel` creates divergence as it's being passed to a `justifyContent` prop. Best to keep things exactly 1:1.